<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChaKanLiuShuiZhang</class>
 <widget class="QWidget" name="ChaKanLiuShuiZhang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>792</width>
        <height>649</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox_kx">
           <property name="currentText">
            <string>款项</string>
           </property>
           <item>
            <property name="text">
             <string>款项</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>收入</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>支出</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_jzy">
           <property name="currentText">
            <string>记账员</string>
           </property>
           <item>
            <property name="text">
             <string>记账员</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_status">
           <item>
            <property name="text">
             <string>状态</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>待确认</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>已确认</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>从</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_datebegin"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>到</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_dateend"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>马上搜索</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>首页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>前页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>后页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>10</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>尾页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>页次:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_page">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>页次:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>30条/页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_count">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>共条信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>转到</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_page">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <family>Aharoni</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>GO</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>tableView</zorder>
      <zorder></zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>freshA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>654</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>FirstPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>637</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>FrontPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>593</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>nextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>LastPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>538</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>GoPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>754</x>
     <y>636</y>
    </hint>
    <hint type="destinationlabel">
     <x>807</x>
     <y>642</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ChaKanLiuShuiZhang</receiver>
   <slot>ConditionalSearch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>freshA()</slot>
  <slot>FirstPage()</slot>
  <slot>FrontPage()</slot>
  <slot>nextPage()</slot>
  <slot>LastPage()</slot>
  <slot>GoPage()</slot>
  <slot>ConditionalSearch()</slot>
 </slots>
</ui>
